Clicker/View Controllers/PollBuilderViewController.swift

```diff
@@ -246,17 +246,17 @@ class PollBuilderViewController: UIViewController, QuestionDelegate, PollBuilder
         print("start question")
         
         // MULTIPLE CHOICE
-        if (questionType == "MULTIPLE_CHOICE") {
+        if (questionType == MULTIPLE_CHOICE) {
 
             let question = mcPollBuilder.questionTextField.text
             let options = mcPollBuilder.optionsDict.keys.sorted().map { mcPollBuilder.optionsDict[$0]! }
             
-            startPollDelegate.startPoll(text: question!, type: "MULTIPLE_CHOICE", options: options, isShared: false)
+            startPollDelegate.startPoll(text: question!, type: MULTIPLE_CHOICE, options: options, isShared: false)
```

Comment:
don't force unwrap these. At the minimum do `let question = mcPollBuilder.questionTextField.text ?? ""`, but think about the best option here